tests/test_biophysical/test_animal/test_cow_calf/test_beef_herd.py

```
@@ -0,0 +1,775 @@
+        vertical_dist_to_milking_parlor=0.0,
+        horizontal_dist_to_milking_parlor=0.0,
+        number_of_stalls=10,
+        housing_type="freestall",
+        pen_type="freestall",
+        animal_combination=AnimalCombination.LAC_COW,
+        max_stocking_density=1.0,
+        minutes_away_for_milking=0,
+        first_parlor_processor=None,
+        parlor_stream_name=None,
+        manure_streams=[{"stream_name": "s1", "stream_proportion": 1.0, "bedding_name": "bedding_1"}],
+    )
```

Ruff (0.15.20)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)